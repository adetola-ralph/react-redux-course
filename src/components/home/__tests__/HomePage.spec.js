import { create } from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
const { mount } = Enzyme;

import HomePage from '../HomePage';

let wrapper;

describe('Home page component', () => {
  // TODO: test this more extensively
  // beforeAll(() => {
  //   wrapper = mount(<HomePage />);
  // });

  test('Snapshot', () => {
    const tree = create(
      <MemoryRouter>
        <HomePage />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
