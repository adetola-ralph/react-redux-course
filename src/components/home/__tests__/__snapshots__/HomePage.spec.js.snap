// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page component Snapshot 1`] = `
<div
  className="jumbotron"
>
  <h1>
    Pluralsight Administration
  </h1>
  <p>
    React, Redux and React Router in ES6 for ultra responsive web apps.
  </p>
  <a
    className="btn btn-primary btn-lg"
    href="/about"
    onClick={[Function]}
  >
    Learn More
  </a>
</div>
`;
